tests/bsim/bluetooth/audio/src/cap_commander_test.c

```diff
@@ -598,13 +1072,57 @@ static void test_main_cap_commander_capture_and_render(void)
 	PASS("CAP commander capture and rendering passed\n");
 }
 
+static void test_main_cap_commander_broadcast_reception(void)
+{
+	size_t accepter_count;
+
+	accepter_count = get_dev_cnt() - 2;
```

Comment:
Please add a comment why this is the case (I assume 1 device is a the CAP commander and the other is a broadcast source)